### PR TITLE
DEVELOPER-1352: Updated Fuse get-started to new style content

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by paulrobinson** [comment]
retest this please

File: products/fuse/get-started.adoc [hunk 1]
```diff
@@ -1,48 +1,116 @@
...
+*redhat-techpreview-all-repository* profile and then click Ok. +
 +
-      esb:create-admin-user
+[.content-img]
+image:#{cdn(site.base_url + '/images/products/devstudio/images_products_devstudio_devstudio-overview-1.png')}[Add Maven Repository]
+d.  Click Finish +
 +
-And type in a username and password.
+[.content-img]
+image:#{cdn(site.base_url + '/images/products/devstudio/images_products_devstudio_devstudio-overview-2.png')}[Configure
```
Comment (LightGuard) [inline products/fuse/get-started.adoc]: This line and the next line are on two lines, separating the title capture (the text between `[` and `]`). This is breaking the build.

**Comment by LightGuard** [comment]
Looks like this last failure was unrelated to this change.

**Comment by LightGuard** [comment]
retest this please

**Pull request merged**